itenium-socks/src/app/socks/shop.component.html

```diff
@@ -14,7 +14,7 @@ <h2>
             </div>
             <div class="detail-box">
               <h6>{{ sock.name }}</h6>
-              <h6><span>{{ sock.price }}</span></h6>
+              <h6><span>{{ sock.price | currency: 'EUR':'symbol':'1.2-2'}}</span></h6>
```

Comment:
Op zich is dit dan allemaal niet meer nodig? En kan je gewoon doen ` | currency`